railgun/manager/manager.go

```diff
@@ -145,6 +146,7 @@ Kong's Admin SSL certificate.`)
 		`add a header (key:value) to every Admin API call, this flag can be used multiple times to specify multiple headers`)
 	flagSet.StringVar(&c.KongAdminToken, "kong-admin-token", "",
 		`The Kong Enterprise RBAC token used by the controller.`)
+	flagSet.BoolVar(&c.EnableReverseSync, "enable-reverse-sync", false, `Enable reverse checks from Kong to Kubernetes`)
```

Comment:
Am I the only one who finds this description very misleading?

To my understanding, "reverse sync" is as much as "always sync, ignore the SHA check"

Reply:
Nah, the name isn't great, but it should remain since it's an existing 1.x flag. We should add another flag, make the check an or condition, and mark this one deprecated if we want to rename it.

Reply:
This was the original 1.x language, as @rainest stated. I'm OK with changing it however if you please provide a github suggestion for the language you prefer? and ofc travis needs to be cool with it too

Reply:
I agree that the flag name should remain, but the description is not useful IMO. And my original comment was meant to suggest improved description only.

Reply:
The suggestion button does not work. I would phrase this as `Send configuration to Kong even if the configuration checksum has not changed since previous update`

Reply:
Updated to the new description as per b9572dd8814b7a8139cea7bec769166790fbac23 :+1: